seshat-node/native/Cargo.toml

```diff
@@ -14,11 +14,11 @@ crate-type = ["cdylib"]
 # introducing breaking changes in patch releases.
 
 [dependencies]
-fs_extra = "1.2.0"
+fs_extra = "1.3.0"
 serde_json = "1.0.61"
 neon-serde2 = "0.8.0"
-uuid = "0.8.2"
-seshat = "2.3.3"
+uuid = "1.3.0"
+seshat = { path = "../../", version = "2.3.3" }
```

Comment:
I've bump it to 2.3.4 I guess it should match ?

Reply:
Once it's published, yes. :)

Reply:
To expand on this, we're currently using `path` to check that it compiles with the current version. The thing we specify as `version` only looks at things that were published.